build

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env sh
+set -eux
```

Comment:
Do we want `-eux` instead of `-e`?

Reply:
Sure, it tells use what's going on and prevents unset variables